data/mods/Magiclysm/worldgen/forest_tomb.json

```diff
@@ -0,0 +1,85 @@
+[
+  {
+    "type": "palette",
+    "id": "tomb",
+    "terrain": {
+      ".": [ "t_rock_floor" ],
+      " ": [ [ "t_grass_dead", 2 ], [ "t_dirt", 15 ], [ "t_dirtmound", 3 ], [ "t_rock_floor_no_roof", 8 ] ],
+      "T": [ "t_tree_dead" ],
+      "S": [ "t_shrub" ],
+      "<": [ "t_slope_up" ],
+      ">": [ "t_slope_down" ],
+      "#": [ "t_rock" ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "forest_tomb" ],
+    "//": "Ground level - Z-0.",
+    "weight": 100,
+    "object": {
+      "rows": [
+        "                  S     ",
+        "            S           ",
+        "   S                    ",
+        "                    S   ",
+        "        S               ",
+        "   S            S       ",
+        "   ##################   ",
+        "   ##################   ",
+        " S ####..##>>##...###   ",
+        "   ###$..........####S  ",
```

Comment:
Also can't find x and $ in definitions.  Are they monsters?

Reply:
$ is weaker monsters.
x is loot.
I will probably reverse these two.